docs/syntax-reference.md

```diff
@@ -290,6 +290,24 @@ In the above statements from_item is one of the following:
 - `table_name [ [ AS ] alias]`
 - `from_item LEFT JOIN from_item ON join_condition`
 
+The WHERE clause may refer to any column defined for a stream or table, including the two implicit columns `ROWTIME`
+and `ROWKEY`.
+
+Example:
+
+```sql
+SELECT * FROM pageviews
+  WHERE ROWTIME >= 1510923225000
+    AND ROWTIME <= 1510923228000;
+```
+
+> Tip: If you are interested in selecting past data, you may need to configure KSQL to query the stream from the very
```

Comment:
`Tip: If you want to select older data, you can configure KSQL to query the stream from the beginning, before the query was run:`